### PR TITLE
Ios/3.12.0

### PR DESCRIPTION
**Pull request by mbh-mapspeople**

### CODE REVIEW

File: collections/_ios/v3/map-styling/index.md [hunk 1]
```diff
@@ -21,6 +21,10 @@ Each has its own purpose which will be explained below.
 
 In the [MapsIndoors CMS](https://cms.mapsindoors.com/types) you can set display settings for the different types of locations in your MapsIndoors content. The changes you make in the CMS will take effect whenever your app reboots or when you call `MapsIndoors.synchroniseContent()` within the app session.
 
+A Display Rule encapsulates both what, how and when a Location should be displayed on the map.
+A Location is presented on the map using a combination of icon, text and polygon. Each of these can appear at different, independent ranges of zoomlevels.
+For example a venue can appear as a marker-icon on low zoomlevels, when zooming in the venuename can appear, and zooming even more in the venua polygon can appear.
```
Comment (Daniel-Nielsen) [inline collections/_ios/v3/map-styling/index.md:26]:
```suggestion
For example a venue can appear as a marker-icon on low zoomlevels, when zooming in the venuename can appear, and zooming even more in the venue polygon can appear.
```

**Review by Daniel-Nielsen — CHANGES_REQUESTED** [review]

**Review by Daniel-Nielsen — APPROVED** [review]

**Comment by marksdk** [comment]
@mbh-mapspeople Don't forget to merge this one in if there are no conflicts with `main` 🙂

**Comment by Daniel-Nielsen** [comment]
@marksdk, if I am not mistaken, this branch is waiting for the actual feature to be released.

**Comment by Daniel-Nielsen** [comment]
Also, the version has been hijacked by the Live Data feature...

**Pull request closed without merging**